Remotion/Web/Core/UI/Controls/DefaultFallbackNavigationUrlProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using Remotion.ServiceLocation;
+
+namespace Remotion.Web.UI.Controls
+{
+  /// <summary>
+  /// Implements <see cref="IFallbackNavigationUrlProvider"/> interface with a default URL of "#".
+  /// </summary>
+  /// <threadsafety static="true" instance="true" />
+  [ImplementationFor(typeof(IFallbackNavigationUrlProvider), Lifetime = LifetimeKind.Singleton, Position = 1)]
```

Comment:
We actually do not need the Position-attribute since our own production code doesn't have alternatives, but if you do keep it to make overrding this in downstream projects eaiser/more deterministic, please check the pattern:

Add const field to the type:
```
    public const int Position = 0;
```

Use the const in the attribute:
```
[ImplementationFor(typeof(IFallbackNavigationUrlProvider), Lifetime = LifetimeKind.Singleton, Position = Position)]
```

This way, it allows an override by referencing the base value and adding an offset. 

Also, the base value is 0, not 1.